File: app/src/main/java/ir/hosseinabbasi/payconiq/data/db/model/License.java
package ir.hosseinabbasi.payconiq.data.db.model;

import javax.annotation.Generated;
import com.google.gson.annotations.SerializedName;

import io.realm.RealmObject;

@Generated("com.robohorse.robopojogenerator")
public class License extends RealmObject{

	@SerializedName("name")
	private String name;

	@SerializedName("spdx_id")
	private String spdxId;

	@SerializedName("key")
	private String key;

	@SerializedName("url")
	private String url;

	public void setName(String name){
		this.name = name;
	}

	public String getName(){
		return name;
	}

	public void setSpdxId(String spdxId){
		this.spdxId = spdxId;
	}

	public String getSpdxId(){
		return spdxId;
	}

	public void setKey(String key){
		this.key = key;
	}

	public String getKey(){
		return key;
	}

	public void setUrl(String url){
		this.url = url;
	}

	public String getUrl(){
		return url;
	}

	@Override
 	public String toString(){
		return 
			"License{" + 
			"name = '" + name + '\'' + 
			",spdx_id = '" + spdxId + '\'' + 
			",key = '" + key + '\'' + 
			",url = '" + url + '\'' + 
			"}";
		}
}

File: app/src/main/kotlin/ir/hosseinabbasi/payconiq/data/db/model/License.kt
package ir.hosseinabbasi.payconiq.data.db.model

import javax.annotation.Generated
import com.google.gson.annotations.SerializedName

import io.realm.RealmObject

@Generated("com.robohorse.robopojogenerator")
open class License : RealmObject() {

    @SerializedName("name")
    var name: String? = null

    @SerializedName("spdx_id")
    var spdxId: String? = null

    @SerializedName("key")
    var key: String? = null

    @SerializedName("url")
    var url: String? = null

    override fun toString(): String {
        return "License{" +
                "name = '" + name + '\''.toString() +
                ",spdx_id = '" + spdxId + '\''.toString() +
                ",key = '" + key + '\''.toString() +
                ",url = '" + url + '\''.toString() +
                "}"
    }
}